### PR TITLE
H.E.C.K. Suit and drake armour cover clothes [WIP]

### PR DESCRIPTION
**Pull request by 4dplanner**
:cl:
fix: HECK suits and drake armor cover clothes, preventing them from burning
/:cl:

Currently these fireproof suits protect your body but not your clothes, which doesn't really make sense.
Todo: check whether coverage should be on parent type of HECK suit instead.
Todo: decide whether flags_inv controlling both transparency and fire makes sense

### CODE REVIEW

**Comment by Daxxed** [comment]
Fixing the "fire fixes" at last.
Quite sad that other people have to clean up the mess.
Many people even mentioned it on the orignal PR.

**Comment by davethwave** [comment]
It was never fixed however since the PR creator explained some kind of twisted logic to make sense of the bug rather that fix it...

**Comment by 4dplanner** [comment]
I'll make a separate PR to fix #42005 later , by the way

**Comment by 4dplanner** [comment]
I've got a better fix, will code later

File: code/modules/clothing/suits/cloaks.dm [hunk 1]
```diff
@@ -71,7 +71,7 @@
 	icon_state = "golhood"
 	desc = "A protective & concealing hood."
 	armor = list("melee" = 35, "bullet" = 10, "laser" = 25, "energy" = 10, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 60, "acid" = 60)
-	flags_inv = HIDEEARS|HIDEEYES|HIDEHAIR|HIDEFACIALHAIR
```
Comment (michnia) [inline code/modules/clothing/suits/cloaks.dm]: 1

Comment (4dplanner) [reply code/modules/clothing/suits/cloaks.dm]: This is a closed PR.

**Pull request closed without merging**